project/config/pytest.ini

```diff
@@ -11,5 +11,6 @@ python_files =
   tests.py
 addopts =
   --cov
-  --cov-append
```

Comment:
We don't need to append anymore. It was hackish anyway. For each Python version, the test run generates different coverage file. After every run, we can combine all coverage files into one. This is much cleaner!